scipy/stats/_entropy.py

```diff
@@ -48,6 +49,42 @@ def entropy(pk: np.typing.ArrayLike,
     S : {float, array_like}
         The calculated entropy.
 
+    Notes
+    -----
+    Informally, the Shannon entropy quantifies the expected uncertainty
+    inherent in the possible outcomes of a discrete random variable.
+
+    If messages consisting of sequences of symbols from a set S are to be
+    encoded and transmitted over a noiseless channel, then the Shannon entropy
+    ``H(pk)`` gives a tight lower bound for the average number of units of
+    information needed per symbol if the symbols occur with frequencies
+    governed by the discrete distribution `pk` [1]_. The choice of base
+    determines the choice of units, ``e`` for nats, ``2`` for bits, etc.
+
+    A related quantity, the cross entropy ``H(pk, qk)``, gives the average
```

Comment:
This makes the relationships much more clear, but it is making me wish that `entropy(pk, qk)` calculated the cross entropy instead of the KL divergence.
The same symbol `H` is used for entropy `H(pk)` and cross entropy `H(pk, qk)`, but `entropy(pk, qk)` is the KL divergence instead of the cross-entropy.
Is there a way to phrase this so that the reader would tend to agree with the choice that `H(pk, qk)` returns KL divergence instead of cross entropy?

Reply:
It's a good point. That cross entropy and Shannon entropy commonly share the symbol `H` does make the current behavior seem at least a little odd. My impression is that relative entropy appears in more contexts and is generally more useful than cross entropy, so that's maybe what the original author had in mind. I don't have any suggestions at the moment beyond stating that KL divergence is used because of historical decisions. I don't feel confident to make any kind of judgement that KL divergence is the more useful quantity and try to use that to justify the current behavior.

Reply:
How about this - could you:

- keep the first two paragraphs as they are
- Make the third paragraph about the "relative entropy (also known as KL-divergence" (to make the "relative entropy" name more prominent)
- Use for cross entropy any symbol other than ``H`` that is sometimes found in the literature, and define it last as the sum of the entropy and relative entropy

?

Reply:
Yeah, that sounds good.

Reply:
Let me know what you think of the update. I couldn't find any suitable alternative notations for cross entropy in the literature, I think I've seen `CE` used in slides somewhere before so I've just used that.  I did find complaints about the standard cross entropy notation `H(X, Y)` since the same same exact notation is also used for joint entropy and one must disambiguate based on context. See [here](https://stats.stackexchange.com/questions/373098/difference-of-notation-between-cross-entropy-and-joint-entropy) for instance.